### PR TITLE
Unclear unexpected error in enable offer button in view page as a company

### LINKED ISSUE #139: Unclear unexpected error in enable offer button in view page as a company
When trying to enable an offer that causes max concurrent offers error, it throws an unexpected error.
It should be clear why the offer was not able to be enabled. This problem will also probably occur when an offer was blocked by an admin

### PR DESCRIPTION
**Pull request by kiko-serra**
Closes #139

### CODE REVIEW

File: src/utils/offer/OfferUtils.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ const HumanReadableErrors = Object.freeze({
     "must-be-admin": "Only admin users can do this action.",
     "login-required": "You must be logged in to do this action.",
     "must-be-company": "You must be a company to do this action.",
+    "max-concurrent-offers-reached:5": "You cannot have more than 5 concurrent visible offers."
```
Comment (TiagooGomess) [inline src/utils/offer/OfferUtils.js:10]:
```suggestion
    "max-concurrent-offers-reached:5": "You cannot have more than 5 concurrent visible offers.",
```

File: src/utils/offer/OfferUtils.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ const HumanReadableErrors = Object.freeze({
...
 });
 
 export const getHumanError = (error) => HumanReadableErrors[error] || "An error occurred, please try again.";
+
```
Comment (TiagooGomess) [inline src/utils/offer/OfferUtils.js:14]:
```suggestion

```

**Review by TiagooGomess — CHANGES_REQUESTED** [review]
To solve these linting issues automatically, you should install and enable eslint in your VSCode.

File: src/utils/offer/OfferUtils.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ const HumanReadableErrors = Object.freeze({
     "must-be-admin": "Only admin users can do this action.",
     "login-required": "You must be logged in to do this action.",
     "must-be-company": "You must be a company to do this action.",
+    "max-concurrent-offers-reached:5": "You cannot have more than 5 concurrent visible offers.",
```
Comment (DoStini) [inline src/utils/offer/OfferUtils.js:10]: The solution is not as simple as this, you shouldn't hard code the value `5`. 
You can take a look at how it's being done in [other places](https://github.com/NIAEFEUP/nijobs-fe/blob/d6d0d2fd0f08c6600857259112cc6dc11ea27067/src/components/Offers/New/CreateOfferUtils.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L51-L55).

**Review by DoStini — CHANGES_REQUESTED** [review]

**Comment by TiagooGomess** [comment]
Closing this because it's a duplicate of #194

**Pull request closed without merging**